proposals/sip_29_listing_cbBTC_and_EURC/description.md

```diff
@@ -0,0 +1,57 @@
+# [SIP-29] Listing cbBTC and EURC
+
+## Summary
+
+The Seamless community would like to add cbBTC and EURC to Seamless lending markets as collateral and borrowable assets.
+
+## Context and motivation
+
+Both cbBTC and EURC are new and important assets on Base and should be listed on Seamless Protocol.
+
+### Chaos Labs Recommendations
+
+**cbBTC**
+
+|Parameter | Value (Base)|
+|--|--|
+|Isolation Mode | No|
+|Borrowable | Yes|
+|Collateral Enabled | Yes|
+|Supply Cap | 200|
+|Borrow Cap | 20|
+|Debt Ceiling | -|
+|LTV | 73%|
+|LT | 78%|
+|Liquidation Bonus | 10%|
+|Liquidation Protocl Fee | 10%|
+|Variable Base | 0%|
+|Variable Slope1 | 4%|
+|Variable Slope2 | 300%|
+|Uoptimal | 45%|
+|Reserve Factor | 20%|
+
+**EURC**
+
+|Parameter| Value|
```

Comment:
two tables are differently formatted (different order of rows, X% vs X.00%, header names, etc.)